core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -58,7 +58,10 @@ import scala.util.{Failure, Success, Try}
  *            enable.auto.commit=false
  *       3. Mirror Maker Setting:
  *            abort.on.send.failure=true
+ *
+ * @deprecated Since 3.0, use the Connect-based MirrorMaker instead (aka MM2).
  */
+@Deprecated
```

Comment:
Since this is a Scala class, you should use the scala deprecated annotation.

Reply:
ah, I found a mix in use, but happy to switch.